src/ServiceInsight/Models/StoredMessage.cs

```diff
@@ -19,9 +19,7 @@ public StoredMessage()
         public MessageIntent MessageIntent { get; set; }
         public Endpoint SendingEndpoint { get; set; }
         public Endpoint ReceivingEndpoint { get; set; }
-        public TimeSpan CriticalTime { get; set; }
         public TimeSpan ProcessingTime { get; set; }
-        public TimeSpan DeliveryTime { get; set; }
         public string ConversationId { get; set; }
 
         public MessageStatistics Statistics
```

Comment:
@distantcam I think that this class is not used anywhere at the moment, can we get rid of it or did I miss something?